test/unit/memcached_test.rb

```diff
@@ -22,6 +22,7 @@ def setup
       :distribution => :modula,
       :show_backtraces => true}
     @cache = Memcached.new(@servers, @options)
+    @cache.flush
```

Comment:
why is this needed? precaution in case there is garbage on the backend?

Reply:
Because I was running into leaky tests. They predate this PR, I just fixed it there.